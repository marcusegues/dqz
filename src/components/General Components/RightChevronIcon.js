import React from 'react';
import { View } from 'react-native';
import Entypo from '@expo/vector-icons/Entypo';
import { moderateScale } from '../../styles/Scaling';

const RightChevronIcon = () => (
  <View style={{ alignSelf: 'center' }}>
    <Entypo name="chevron-right" size={moderateScale(25)} color="#24253D" />
  </View>
);

export default RightChevronIcon;
