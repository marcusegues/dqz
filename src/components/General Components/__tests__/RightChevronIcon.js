import React from 'react';
import renderer from 'react-test-renderer';
import { RightChevronIcon } from '../RightChevronIcon';

describe('RightChevronIcon', () => {
  test('renders according to snapshot', () => {
    const component = renderer.create(<RightChevronIcon />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
