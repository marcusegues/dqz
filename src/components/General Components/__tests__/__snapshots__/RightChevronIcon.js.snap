// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RightChevronIcon renders according to snapshot 1`] = `
<View
  style={
    Object {
      "alignSelf": "center",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  />
</View>
`;
